apps/condo/domains/property/components/PropertyImport.tsx

```diff
@@ -0,0 +1,215 @@
+        okButtonProps: {
+            type: 'primary',
+            danger: true,
+        },
+    }
+}
+
+const getPropertyUploadSuccessModalConfig = () => {
+    return {
+        // TODO(Dimitreee):add translations
+        title: 'Импорт',
+        closable: true,
+        content: (
+            <Alert
+                style={{ marginTop: 16 }}
+                // TODO(Dimitreee): add translations
+                message='Дома добавлены.'
+                type='success'
+            />
+        ),
+        okText: 'Продолжить',
+    }
+}
+
+const getPropertyUploadErrorModalConfig = () => {
+    return {
+        // TODO(Dimitreee):add translations
+        title: 'Импорт',
+        closable: true,
+        content: (
+            <ModalContext.Consumer>
+                {
+                    ({ error }) => {
+                        const errorMessage = get(error, 'message') || 'Во время обработки документа произошла ошибка, проверьте корректность предоставленных данных или обратитесь в службу поддержки.'
+
+                        return (
+                            <Alert
+                                style={{ marginTop: 16 }}
+                                // TODO(Dimitreee): add translations
+                                message={errorMessage}
+                                type='error'
+                            />
+                        )
+                    }
+                }
+            </ModalContext.Consumer>
+        ),
+        okText: 'Продолжить',
+    }
+}
+
+interface IPropertyImport {
+    onFinish(): boolean
+}
+
+export const PropertyImport: React.FC<IPropertyImport> = (props) => {
+    const userOrganization = useOrganization()
+    const auth = useAuth()
+    const [modal, contextHolder] = Modal.useModal()
+    const activeModal = useRef(null)
+
+    const destroyActiveModal = () => {
+        if (activeModal.current) {
+            activeModal.current.destroy()
+            activeModal.current = null
+        }
+    }
+
+    const [importData, progress, error, isImported, breakImport] = useImporter(
+        () => {
+            destroyActiveModal()
+            const config = getPropertyUploadSuccessModalConfig()
+            activeModal.current = modal.success(config)
+
+            props.onFinish()
+        },
+        () => {
+            destroyActiveModal()
+            const config = getPropertyUploadErrorModalConfig()
+            activeModal.current = modal.error(config)
+        },
+    )
+
+    const handleUpload = useCallback((file) => {
+        destroyActiveModal()
+        const config = getPropertyUploadInfoModalConfig(() => {
+            breakImport()
+            props.onFinish()}
+        )
+        activeModal.current = modal.info(config)
+
+        importData(file.data)
+    }, [])
+
+    const organizationCreatorId = get(userOrganization, ['organization', 'createdBy', 'id'])
+    const userID = get(auth, ['user', 'id'])
+    const isOwner = organizationCreatorId === userID
```

Comment:
It's not true. You should check the Role permissions here. Employee.role.canManageProperties

Reply:
going to use isAdmin